service/reportsStorage/handlers.go

```diff
@@ -0,0 +1,188 @@
+package reportsStorage
+
+import (
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/reportsStorage/models"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/reportsStorage/models (cannot find package "github.com/iryonetwork/wwm/gen/reportsStorage/models" in any of:
	/tmp/tmp.jNUNwZeI6Z/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/reportsStorage/models (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/reportsStorage/models (from $GOROOT)
	/tmp/tmp.jNUNwZeI6Z/src/github.com/iryonetwork/wwm/gen/reportsStorage/models (from $GOPATH))